topologies/kne/arista_ixia.textproto

```diff
@@ -39,7 +39,7 @@ nodes: {
         config_path: "/mnt/flash"
         config_file: "startup-config"
         file: "arista_ceos.config"
-    	image: "us-central1-docker.pkg.dev/kt-nts-athena-dev/keysight/ceos:4.29.0F-28265717"
+    	image: "us-central1-docker.pkg.dev/kt-nts-athena-dev/keysight/ceos:4.29.0F-28493660"
```

Comment:
we should never check images from private repos

this should just be "ceos:latest"

and it is then up to the runner of this to either provide that image or substitute it

Reply:
I will change to "latest"